src/main/app/claims/models/claim.ts

```diff
@@ -532,7 +534,12 @@ export class Claim {
   }
 
   private hasClaimantRejectedPartAdmission (): boolean {
-    return this.claimantResponse && this.claimantResponse.type === ClaimantResponseType.REJECTION
+    return !ClaimFeatureToggles.isFeatureEnabledOnClaim(this, 'directionsQuestionnaire') && this.claimantResponse && this.claimantResponse.type === ClaimantResponseType.REJECTION
+      && this.response.responseType === ResponseType.PART_ADMISSION
+  }
+
+  private hasClaimantRejectedPartAdmissionDQs (): boolean {
+    return ClaimFeatureToggles.isFeatureEnabledOnClaim(this, 'directionsQuestionnaire') && this.claimantResponse && this.claimantResponse.type === ClaimantResponseType.REJECTION
```

Comment:
above two methods are same except one returns true and other false. Can we put both in 1 method (if-else-block) and change method name to something more appropriate like `hasClaimantRejectedPartAdmissionWithDQ` , just a suggestion!!

Reply:
I'm not sure I understand your point? can you come explain what you mean to me?

Reply:
i guess its not an issue if we are removing this logic when DQ is enabled in Production.